lib/log/sfWebDebugLogger.class.php

```diff
@@ -76,11 +75,6 @@ public function handlePhpError($errno, $errstr, $errfile, $errline, $errcontext
         $message = sprintf(' %%s at %s on line %s (%s)', $errfile, $errline, str_replace('%', '%%', $errstr));
 
         switch ($errno) {
-            case E_STRICT:
```

Comment:
E_STRICT is deprecated since PHP 5.4; as our lowest supported version is now 7.4, we can simply drop E_STRICT.